src/styles/blocks/input.css

```
@@ -0,0 +1,81 @@
+.input {
+  position: relative;
+}
+
+.input::before {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  content: '';
+  display: block;
+  background-image: url(../../images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.input--big::before {
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input--small::before {
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input__field {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  display: block;
+  width: 100%;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8 rgba(61, 78, 97, 0.1);
+}
+
+.input__field:hover {
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+}
+
+.input__field:focus {
+  font-weight: bold;
+  color: #3D4E61;
+  background: linear-gradient(#FFF, #F6F6F7);
```

Move color to input - it will fix place holder 
```suggestion
  font-weight: 300;
  display: block;
  width: 100%;
  color: #3D4E61;
  border: 1px solid #E1E7ED;
  border-radius: 4px;
  box-shadow: 0 1px 8 rgba(61, 78, 97, 0.1);
}

.input__field:hover {
  border: 1px solid #E1E7ED;
  border-radius: 4px;
}

.input__field:focus {
  font-weight: bold;
  background: linear-gradient(#FFF, #F6F6F7);
```